contracts/contracts/vault/VaultCore.sol

```diff
@@ -185,7 +186,7 @@ contract VaultCore is VaultStorage {
 
         if (_minimumUnitAmount > 0) {
             uint256 unitTotal = 0;
-            for (uint256 i = 0; i < outputs.length; i++) {
+            for (uint256 i = 0; i < outputs.length; ++i) {
```

Comment:
Is there no gas benefits to having the length broken out as a separate variable? `outputs` is memory, but a stack item should still be faster than a memory item? Or does this stack too deep?

Reply:
Not in this case as `outputs` is an in memory array.

The other cases were storage arrays and will do an SLOAD to read the array length on each loop.

Reply:
Looks like it moves from 233,740 to 233,736 when I change it in the OUSD tests. That's fairly minor.